integration/test/utils/http_utils.go

```diff
@@ -0,0 +1,20 @@
+package utils
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+func GetResponse(url string) (string, error) {
+	resp, err := http.Get(url)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	sb := string(body)
+	return sb, err
```

Comment:
better to make the err clear in the return so the reader isn't forced to trace all previous code to determine what err may be at the return site

`return sb, nil`
or remove the previous line
`return string(body), nil`